code/modules/projectiles/ammunition/ballistic/rifle.dm

```diff
@@ -10,7 +10,7 @@
 /obj/item/ammo_casing/a762/enchanted
 	projectile_type = /obj/item/projectile/bullet/a762_enchanted
 
-// 5.56mm (M-90gl Carbine)
+// 5.56mm (Unused)

```

Comment:
The `NT-ARG 'Boarder'` and `X9 Assault Rifle` should still be using this caliber, so please remove the "Unused" bit, it's misleading.